core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -46,28 +47,19 @@ export class ContributorAdminDashboardPageComponent implements OnInit {
   TAB_NAME_QUESTION_REVIEWER: string = 'Question Reviewer';
   translationReviewersCount: number = 0;
   questionReviewersCount: number = 0;
-  CONTRIBUTION_TYPES: string[] = [this.TAB_NAME_TRANSLATION_SUBMITTER,
-    this.TAB_NAME_TRANSLATION_REVIEWER,
-    this.TAB_NAME_QUESTION_SUBMITTER,
-    this.TAB_NAME_QUESTION_REVIEWER];
+  CONTRIBUTION_TYPES!: string[];
+  selection!: string;
```

Comment:
Can you please explain what is this selection variable for ?

Reply:
Got it thanks, but then isn't selection a bit vague ? It's not conveying what's being selected, probably something like `selectedContributionType` would be more accurate, same for the `updateSelection`.

Reply:
Sg. I have changed both things please take a look. Thanls.